tests/server/endpoints/test_orders_endpoint.py

```diff
@@ -39,3 +40,25 @@ def test_orders_endpoint(
 
     # THEN the response contains the correct number of orders
     assert len(response.json["orders"]) == expected_orders
+
+
+def test_order_endpoint(
+    client: FlaskClient,
+    order: Order,
+    order_another: Order,
+    order_balsamic: Order,
```

Comment:
The `order_another` and `order_balsamic` are not used? I don't think it is necessary to test all of them

Reply:
I added them, but probably only one additional is necessary. I wanted to make sure that we did not fetch all orders which required more than the one fetched to be present.